support/developer/biztalk/management-operations/maintain-troubleshoot-database.md

```diff
@@ -273,22 +275,22 @@ Additionally, use the PSSDiag utility to collect data on the `Lock:Deadlock` eve
 
 The `BizTalkMsgBoxDB` database is a high-volume and high-transaction Online Transaction Processing (OLTP) database. Some deadlocking is expected, and this deadlocking is handled internally by the BizTalk Server engine. When this behavior occurs, no errors are listed in the error logs. When you investigate a deadlock scenario, the deadlock that you are investigating in the output must be correlated with a deadlock error in the event logs.
 
-The dequeue command and some SQL Server Agent jobs are expected to deadlock. Typically, these jobs are selected as deadlock victims. These jobs will appear in a deadlock trace. However, no errors are listed in the event logs. Therefore, this deadlockling is expected, and you can safely ignore the deadlocking with these jobs.
+The `dequeue` command and some SQL Server Agent jobs are expected to deadlock. Typically, these jobs are selected as deadlock victims. These jobs will appear in a deadlock trace. However, no errors are listed in the event logs. Therefore, this deadlocking is expected, and you can safely ignore the deadlocking with these jobs.
```

Comment:
Have you checked the command? We can't use code mark if it's not a command name.